pkg/clients/aws/route53.go

```diff
@@ -528,3 +528,26 @@ func getSTSCredentials(client *sts.STS, roleArn string, externalID string, roleS
 	}
 	return assumeRoleOutput, nil
 }
+
+func listAllHostedZones(r53 route53iface.Route53API, lhzi *route53.ListHostedZonesInput) ([]*route53.HostedZone, error) {
```

Comment:
To be consistent, can we add a comment to the purpose of the function? Although the name is pretty self-explanatory.

Reply:
done!